### PR TITLE
Fix #17138 - Allow ActionMessage to be autohidden after a given number of milliseconds

### LINKED ISSUE #17138: NFTs:  Toaster notification never goes away naturally
### Describe the bug

The "Collectible was successfully added!" element that displays after importing an NFT never naturally goes away -- it just hangs there.  I would think we'd want that to go away after a given amount of time.

NFTSSSS

Not removing this leads to clutter, and closing it is not an action we should require the user to take.

### Steps to reproduce

1.  Import an NFT
2.  View the toaster modal at the bottom right

### Error messages or log output

_No response_

### Version

10.23

### Build type

None

### Browser

Chrome

### Operating system

MacOS

### Hardware wallet

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by darkwing**
The `ActionMessage` instance that displays on the home screen after a NFT has been added is annoying.  We should hide it after a given amount of time.  Instead of only allowing that functionality for this use case, adding that capability to ActionMessage will allow it to be used everywhere.

* Fixes #17138

## Manual Testing Steps

1. Add a new NFT
2. See that the Actionable message hides after 5 seconds

## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Review by pedronfigueiredo — DISMISSED** [review]

**Review by NidhiKJha — COMMENTED** [review]
LGTM

**Review by NidhiKJha — DISMISSED** [review]

File: ui/pages/home/home.component.js [hunk 1]
```diff
@@ -312,6 +312,7 @@ export default class Home extends PureComponent {
           <ActionableMessage
             type="success"
             className="home__new-network-notification"
+            autoHideTime={5000}
```
Comment (ryanml) [inline ui/pages/home/home.component.js:315]: Nit: could we use the time constants provided by https://github.com/MetaMask/metamask-extension/blob/develop/shared/constants/time.ts here? (`SECOND * 5`)

**Review by NidhiKJha — APPROVED** [review]
LGTM 🚀

**Review by ryanml — APPROVED** [review]
LGTM!

**Pull request merged**